src/includes/performance/traces-sampler-as-sampler/ruby.mdx

```diff
@@ -4,14 +4,20 @@ Sentry.init do |config|
     # transaction_context is the transaction object in hash form
     # keep in mind that sampling happens right after the transaction is initialized
     # e.g. at the beginning of the request
-    transaction = sampling_context[:transaction_context]
+    transaction_context = sampling_context[:transaction_context]
 
-    # if the transaction is important, set a higher rate
-    if transaction[:name].match?("orders")
-      0.5
-    # otherwise, give it a lower rate
-    else
-      0.1
+    # with transaction_context, you can sample transactions more sophisticatedly
```

Comment:
@PeloWriter could you help me refine this example as well? thanks 🙏

Reply:
Sure :)